src/memory_server.ml

```diff
@@ -21,10 +21,31 @@ open Squeezed_state
 open Squeezed_xenstore
 open Xapi_stdext_monadic
 open Xapi_stdext_unix
+open Xapi_stdext_std
+open Xapi_stdext_threads
```

Comment:
Please open this locally in the new module or follow the style of the rest of the file.

Reply:
Yes.